core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1652,9 +1649,14 @@ public function deleteChilds($table, $id, &$delete)
 
 			if ($objDelete->numRows && !($GLOBALS['TL_DCA'][$v]['config']['doNotDeleteRecords'] ?? null) && \strlen($v))
 			{
-				foreach ($objDelete->fetchAllAssoc() as $row)
+				$rows = $objDelete->fetchAllAssoc();
+				$this->preloadCurrentRecords(array_column($rows, 'id'), $table);
```

Comment:
this should use `$v` (the current table name) not `$table`

Reply:
Good catch! Fixed in 8b2e5ca0e0ad333faf0ab84d573fae48ebb7b505.